url_encoder/plugin.spec.yaml

```diff
@@ -0,0 +1,70 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: url_encoder
+title: URL Encoder
+description: Encode and decode URLs
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+cloud_ready: true
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/url_utils
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- url
+- encoder
+- decoder
+- utilities
+hub_tags:
+  use_cases: [data_utility]
+  keywords: [url, encoder, decoder, utilities]
+  features: []
+actions:
+  encode:
+    title: Encoder
+    description: Encode URLs
```

Comment:
Might be a tad more specific here like URL Encode a URL. Same with the Decode action's description

Reply:
How about `description: Percent-encode a URL`?

Is percent too divergent? Trying to avoid a ["how to draw an owl: step 1, draw a circle. step 2, draw the rest of the owl"](https://www.reddit.com/r/restofthefuckingowl/) kind of phrasing.